api/cmd/managerapi.go

```diff
@@ -124,6 +124,24 @@ func NewManagerAPICommand() *cobra.Command {
 				}
 			}()
 
+			// HTTPS
+			if conf.SSLCert != "" && conf.SSLKey != "" {
+				addrSSL := fmt.Sprintf("%s:%d", conf.ServerHost, conf.SSLPort)
+				serverSSL := &http.Server{
+					Addr:         addrSSL,
+					Handler:      r,
+					ReadTimeout:  time.Duration(1000) * time.Millisecond,
+					WriteTimeout: time.Duration(5000) * time.Millisecond,
+				}
```

Comment:
Hey, Nice work :+1: . 
I think we should `go` with adding a custom TLS configuration to make it more secure. Also, a go server automatically upgrades requests to HTTP/2 if a tlsconfig is set with HTTPS.

Referring to an awesome blog from [cloudflare](https://blog.cloudflare.com/exposing-go-on-the-internet/).
Thanks :)

Reply:
updated. thanks.